prober/http_test.go

```diff
@@ -97,6 +101,158 @@ func TestValidHTTPVersion(t *testing.T) {
 	}
 }
 
+func TestContentLength(t *testing.T) {
+	type testdata struct {
+		msg                    string
+		contentLenght          int
+		uncompressedBodyLength int
+		handler                http.HandlerFunc
+	}
+
+	// These test cases cover known encodings (brotli, deflate,
```

Comment:
These tests don't relate to this change.

Reply:
removed